### PR TITLE
fixed Martins comments and small code refactoring

### PR DESCRIPTION
**Pull request by wotd**

### CODE REVIEW

File: cleanup/artifactCleanup/artifactCleanup.groovy [hunk 1]
```diff
@@ -54,7 +56,8 @@ executions {
         def disablePropertiesSupport = params['disablePropertiesSupport'] ? params['disablePropertiesSupport'][0].toBoolean() : false
         Global.paceTimeMS = params['paceTimeMS'] ? params['paceTimeMS'][0] as int : 0
         def keepRelease = params['keepRelease'] ? params['keepRelease'][0].toBoolean() : false
-        def releaseRegex = config.policies[0][7] ? config.policies[0][7] as Pattern : ~/.*-\d+\.\d+\.\d+\.*/
+        def releaseRegex = params['releaseRegex'] ? params['releaseRegex'] as Pattern : regex
+        // def releaseRegex = config.policies[0][7] ? config.policies[0][7] as Pattern : ~/-\d+\.\d+\.\d+/
```
Comment (martinm82) [inline cleanup/artifactCleanup/artifactCleanup.groovy]: You can remove this commented out line

**Review by martinm82 — APPROVED** [review]
LGTM

File: cleanup/artifactCleanup/artifactCleanup.groovy [hunk 2]
```diff
@@ -45,6 +45,8 @@ class Global {
 
 def pluginGroup = 'cleaners'
 def config = new ConfigSlurper().parse(new File(ctx.artifactoryHome.haAwareEtcDir, PROPERTIES_FILE_PATH).toURL())
+def regex = ~/-\d+\.\d+\.\d+/
```
Comment (martinm82) [inline cleanup/artifactCleanup/artifactCleanup.groovy:48]: Could you give as well an example what this regex is used for (semantic versioning).

Comment (wotd) [reply cleanup/artifactCleanup/artifactCleanup.groovy:48]: This regex will keep all artifacts that contains 3 digits version in its name after "-" char. So for example:
art-2.0.0.tgz - will be kept
alpine-20.21.0.tar.gz - will be kept 
alpine-2.tar.gz - will be deleted
alpine-2.0.tar.gz - will be deleted

Comment (anna-irena-nowak) [reply cleanup/artifactCleanup/artifactCleanup.groovy:48]: how does it handle navkit releases like `blah-release-bin-2.0.94-17.1.zip`?

**Review by martinm82 — APPROVED** [review]
LGTM

File: cleanup/artifactCleanup/artifactCleanup.groovy (hunk 2 above)
Comment (wotd) [reply cleanup/artifactCleanup/artifactCleanup.groovy:48]: @wiorka:
`Can't delete docker-local:blah-release-bin-2.0.94-17.1.zip (user admin has no delete permissions), 3/5 total 509116928 bytes`

Comment (martinm82) [reply cleanup/artifactCleanup/artifactCleanup.groovy:48]: This is just an example regex.

**Pull request merged**